styles/colors-cobalt-day.less

```diff
@@ -271,4 +271,4 @@
 
 // Scroller
 // ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
+@agate-scroller-scrollbar-thumb-bg-color: @agate-gray;
```

Comment:
@agate-gray is still hardly visible. Maybe use @agate-accent here as well?

Reply:
Okay